#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

int32 cost

int32 n
int32 ne
int32 e
int32 se
int32 s
int32 sw
int32 w
int32 nw
